### DISCUSSION TITLE
Fooocus 2.5.5Chinese translation【Fooocus 2.X.X 中文翻译文件】

### DISCUSSION THREAD

**Discussion by hotlml** [lllyasviel/Fooocus#2494] [Show and tell]
I have been using the translation files provided by Phrixus2023. #752 However, some modules and the newly released version 2.2.1 have not been updated with Chinese translations. I have updated the Chinese translation files based on Phrixus2023 work. Thanks to Phrixus2023 for his contribution. To use the Chinese translation, simply place the "default" file in the "language" folder.

一直使用 Phrixus2023 提供的翻译文件，但有一些模块和新发布的 2.2.1 版本尚未更新中文翻译。
我基于 Phrixus2023 的工作更新了中文翻译文件。感谢 Phrixus2023 的贡献。
要使用中文翻译，只需将“default”文件放入“language”文件夹即可。

Fooocus 2.2.1 [default.json](https://github.com/lllyasviel/Fooocus/files/14549156/default.json)

Fooocus 2.3.0 [default.json](https://github.com/lllyasviel/Fooocus/files/14643948/default.json)

Fooocus version: 2.5.5 [default.json](https://github.com/user-attachments/files/18879901/default.json)

**Comment by mashb1t** [comment]
Thanks for the update!
It's more versatile to not name the file default.json, but to create the file language/cn.json and use `--language cn` as startup argument.

**Comment by jun1568** [comment]
I followed the prompts, downloaded the file directly and changed it to cn.json, then put it in the language folder, after clicking run.bat to open the software automatically, the Chineseisation was not successful, the interface is still in English, may I ask how to make it take effect?

**Reply by mashb1t** [reply]
please add the argument `--language cn` in run.bat after entry_with_update.py for it to take effect.
You may also make a copy of run.bat, called run-cn.bat or similar.

**Reply by jun1568** [reply]
It has already taken effect as you provided, thank you very much!

**Reply by hotlml** [reply]
It is also possible to use the file without changing the name, but it is better to follow mashb1t method.

**Comment by fish20245** [comment]
2.5.2了老哥，跟新下

**Reply by hotlml** [reply]
> 2.5.2了老哥，跟新下

已经更新到2.5.5了 
"It has been updated to 2.5.5."
